tests/test_ssa.py

```diff
@@ -360,10 +361,32 @@ def f1(x, y):
             a = z
         else:
             a = y
-        a.x = 3
+        if x & y:
+            return a
+        a.x = z
         return a
 
-    f2 = apply_passes([ssa(False)])(f1)
+    class CheckSymbolTable(Pass):
+        def rewrite(self,
+                    tree: cst.FunctionDef,
+                    env: SymbolTable,
+                    metadata: tp.MutableMapping) -> PASS_ARGS_T:
+            assert "ssa_symbol_table" in metadata
+            assert str(metadata["ssa_symbol_table"]) == (
```

Comment:
Why test the serialization and not the contents?

Reply:
Also what is the form of this dict in the end?  `line: int -> name: str: -> ssa_name: str`?

Reply:
I think I was printing the contents for debugging so I just checked the strings once I got what I wanted, testing the contents makes more sense though

Reply:
And yes that's the form the dictionary although it's not necessarily the required format (e.g. we could extract that information from a table mapping the line information)